node/src/types/block.rs

```diff
@@ -1233,6 +1220,17 @@ pub(crate) mod json_compatibility {
     }
 
     impl JsonBlock {
+        /// Create a new JSON Block with a Linear chain block and its associated signatures.
+        pub fn new(block: Block, signatures: BlockSignatures) -> Self {
+            let mut proto_block = JsonBlock::from(block);
```

Comment:
I think the variable would be better named `json_block`, since `proto_block` implies it's a `ProtoBlock` as defined elsewhere in this module.